portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,38 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote from Brooklyn-99 to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ['Toit Nups!', 'NINE-NINE!', 'Cool cool cool cool cool, no doubt no doubt.', 'Why is no one having a good time? I specifically requested it.'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
 }
+
+
+
+/**
+ * This function opens the tab we want to access and hides the unnecessary tabs
+ */
+function openTab(evt, tabName) {
+    var i, tabcontent, tablinks;
```

Comment:
No need to declare each variable at the top like this, we usually just declare each variable as its used.

Reply:
Done